metrics/bigquery.py

```diff
@@ -273,11 +186,6 @@ def main(configs, project, bucket_path, backfill_days):
     PARSER.add_argument(
         '--bucket',
         help='Upload results to the specified gcs bucket.')
-    PARSER.add_argument(
-        '--backfill-days',
-        default=30,
-        type=int,
-        help='Number of days to backfill influxdb data.')
```

Comment:
I think we want to keep this, I would just alter the description to avoid mentioning influxdb.  We're still passing `backfill_days` to main

Reply:
good catch

Reply:
Looks like that is just bing pipped though to BigQuery object and never used. I will clear them.